connectors/spark/src/test/java/io/unitycatalog/spark/TableReadWriteTest.java

```diff
@@ -321,14 +319,19 @@ public void testCreateExternalTableWithoutLocation() {
     SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME);
 
     String fullTableName1 = CATALOG_NAME + "." + SCHEMA_NAME + "." + PARQUET_TABLE;
-    assertThatThrownBy(() -> {
-      session.sql("CREATE EXTERNAL TABLE " + fullTableName1 + "(name STRING) USING parquet");
-    }).hasMessageContaining("Cannot create EXTERNAL TABLE without location");
+    assertThatThrownBy(
```

Comment:
The changes in this file are style changes. Can you remove it from the PR?

Reply:
I can try but testing might fail if the javafmt check fails.